server/bell_oauth_profile.js

```diff
@@ -17,16 +17,16 @@ function profile (credentials, params, get, profileCb) {
     // Bell returns the parsed data and handles errors internally
     log.verbose('exchanged tokens for profile data:' + JSON.stringify(data));
     // TODO use Joi to validate `data` before sending to DB
-    db.getUserById(data.uid, function (err, user) {
+    user.get(data.uid, function (err, user) {
       if (err) {
-        log.warn('getUserById failed: ' + err);
-        return profileCb('getUserById failed: ' + err);
+        log.warn('user.get failed: ' + err);
+        return profileCb('user.get failed: ' + err);
```

Comment:
Hmm, good question. The signature is `profile(err, credentials)`, according to the [bell docs](https://github.com/hapijs/bell). However, looking at the code where [bell calls this function](https://github.com/hapijs/bell/blob/master/lib/oauth.js#L257-L260), it actually doesn't check for an error at all. Kinda funky. This `profile` function doesn't have access to the `reply` or `response` functions...maybe the right thing is to emit an error directly? :-\